ark/docs/components/AutoDocs.tsx

```diff
@@ -0,0 +1,67 @@
+import { flatMorph } from "@ark/util"
+import { ark } from "arktype"
+import { Fragment } from "react"
+
+export const AutoDocs = () => {
```

Comment:
I know it probably doesn't matter because of SSR, but my (probably outdated) intuition is that logic like this that doesn't depend on props should be moved outside the component so it never gets recomputed based on UI changes?

Reply:
yeah it's the same difference since this only runs on the server, but moving the code outside the component is still probably best practice for consistency's sake

Reply:
Yeah, I figured. Good for my own sanity to have the logic work that way if nothing else 😆

I'm so used to thinking about what can be precomputed when and how things are cached from the type system it's hard to kick the habit.